src/deluge/model/instrument/midi_instrument.cpp

```diff
@@ -602,7 +602,8 @@ void MIDIInstrument::noteOnPostArp(int32_t noteCodePostArp, ArpNote* arpNote) {
 			// Count up notes per member channel. This traversal will *not* find the new note that we're switching on, which will have had its toMIDIChannel set to 16 by Arpeggiator (we'll decide and set it below).
 			for (int32_t n = 0; n < arpeggiator.notes.getNumElements(); n++) {
 				ArpNote* thisArpNote = (ArpNote*)arpeggiator.notes.getElementAddress(n);
-				if (thisArpNote->outputMemberChannel >= 1 && thisArpNote->outputMemberChannel <= 14) {
```

Comment:
This seems super right, in fact, the calculation of lowestMemberChannel and highestMemberChannel will give you the REAL member channels configured for each zone, to use them in the comparisons in the code (if blocks and whatnot)